core/app/models/spree/user_address.rb

```diff
@@ -2,7 +2,7 @@
 
 module Spree
   class UserAddress < Spree::Base
-    belongs_to :user, class_name: UserClassHandle.new, foreign_key: "user_id", optional: true
+    belongs_to :user, class_name: UserClassHandle.new, foreign_key: "user_id", optional: true, inverse_of: :user_addresses
     belongs_to :address, class_name: "Spree::Address", optional: true
```

Comment:
Why is this optional? 🤔 This is a join record, this should never have an orphaned address, but should get deleted instead, right?